recipe/meta.yaml

```diff
@@ -1,18 +1,17 @@
 {% set name = "xgboost" %}
-{% set version = "0.6a2" %}
-{% set sha256 = "ebc4e2bf8c8266212e342ff8ec4f6ae469e8c41a05d099b6778de8424ce32563" %}
+{% set version = "0.7" %}
```

Comment:
Set this to `0.7.post3`

Reply:
Yea, I guess it will be less confusing... (the original 0.7 and the following .post1 and .post2 were packaged incorrectly, https://github.com/dmlc/xgboost/issues/2996 -- hence all this)

// There will also likely be a follow-up 0.7.1 release of xgboost at some point soon, with all packaging issues resolved and checked in back to the main codebase.

Reply:
Cool. We can then bump the version when that happens.